invokeai/frontend/web/src/services/api/endpoints/models.ts

```diff
@@ -52,9 +52,18 @@ type UpdateMainModelArg = {
   body: MainModelConfig;
 };
 
+type UpdateLoRAModelArg = {
+  base_model: BaseModelType;
+  model_name: string;
+  body: LoRAModelConfig;
+};
+
 type UpdateMainModelResponse =
   paths['/api/v1/models/{base_model}/{model_type}/{model_name}']['patch']['responses']['200']['content']['application/json'];
 
+type UpdateLoRAModelResponse =
```

Comment:
Could you just alias the type above?

Reply:
Just an optional nit. Literally just assign it to the name above to reduce copy/pasting.